src/plugins/LinkManager/createReactTooltipWrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import Nanobus from "nanobus"
+import React, { ElementType, useEffect, useState } from "react"
+import ReactDOM from "react-dom"
+
+export type TooltipComponentType = string | unknown
```

Comment:
Can this be moved to the types file then be imported in?

Reply:
`TooltipComponentType` is a very interesting type name 😅 Would ideally also be better if we could find out what `unknown` should be.